docs/src/components/HeaderMenu.js

```diff
@@ -1,76 +1,58 @@
-  };
 
   const onChangeColorScheme = () => setColorScheme(colorScheme === 'light' ? 'dark' : 'light');
 
   const onTextDirectionChange = () => setTextDirection(textDirection === 'rtl' ? 'ltr' : 'rtl');
 
   return (
     <Box
+      alignItems="center"
+      color="pine"
       display={isHeader ? 'none' : 'flex'}
       mdDisplay={isHeader ? 'flex' : 'none'}
-      alignItems="center"
       justifyContent={isHeader ? undefined : 'center'}
-      color="pine"
     >
-      <Tooltip inline text={textDirection === 'rtl' ? 'Left-To-Right View' : 'Right-To-Left View'}>
-        <IconButton
-          size="md"
-          accessibilityLabel="toggle page direction: Left-To-Right / Right-To-Left View"
-          iconColor="white"
-          dangerouslySetSvgPath={togglePageDirSvgPath}
-          onClick={() => onTextDirectionChange()}
-        />
-      </Tooltip>
-      <Tooltip inline text={colorScheme === 'light' ? 'Dark-Mode View' : 'Light-Mode View'}>
-        <IconButton
-          size="md"
-          accessibilityLabel="toggle color scheme: light / dark mode views"
-          iconColor="white"
-          icon="workflow-status-in-progress"
-          onClick={() => onChangeColorScheme()}
-        />
-      </Tooltip>
-      <Tooltip
-        inline
-        text={`Sidebar: ${sidebarOrganisedBy === 'categorized' ? 'Alphabetical' : 'Categorize'}`}
-      >
-        <Box display="flex" alignItems="center">
-          <IconButton
-            size="md"
-            accessibilityLabel="Toggle sidebar categorization"
-            iconColor="white"
-            icon={sidebarOrganisedBy === 'categorized' ? 'arrow-circle-down' : 'folder'}
-            onClick={() =>
-              setSidebarOrganizedBy(
-                sidebarOrganisedBy === 'categorized' ? 'alphabetical' : 'categorized',
-              )
-            }
-          />
-        </Box>
-      </Tooltip>
-      <Tooltip inline text="Opens Codesandbox ready to start coding with Gestalt">
+      <LTRButton onClick={() => onTextDirectionChange()} textDirection={textDirection} />
+      <DarkModeButton colorScheme={colorScheme} onClick={() => onChangeColorScheme()} />
+      <SidebarCategorizationButton
+        onClick={() =>
+          setSidebarOrganizedBy(
+            sidebarOrganisedBy === 'categorized' ? 'alphabetical' : 'categorized',
+          )
+        }
+        sidebarOrganisedBy={sidebarOrganisedBy}
+      />
+
+      <Tooltip inline text="Opens CodeSandbox ready to start coding with Gestalt">
         <Text color="white">
-          <GestaltLink href="https://codesandbox.io/s/k5plvp9v8v" target="blank">
+          <GestaltLink
+            href="https://codesandbox.io/s/k5plvp9v8v"
+            onClick={() => trackButtonClick('Playground')}
+            target="blank"
+          >
             <Box padding={2}>Playground</Box>
           </GestaltLink>
         </Text>
       </Tooltip>
+
       <Text color="white">
```

Comment:
Can we change order here?

         <GestaltLink
          href="https://github.com/pinterest/gestalt"
          onClick={() => trackButtonClick('GitHub')}
          target="blank"
        > 
          <Box padding={2}>\
               <Text color="white">GitHub</Text>
         </Box>
        </GestaltLink>

Reply:
One above too.

Reply:
out of scope